system_files/desktop/shared/usr/share/ublue-os/just/86-bazzite-windows.just

```diff
@@ -0,0 +1,9 @@
+# vim: set ft=make :
+
+# Adds a script in Steam to boot Windows which is useful for dual-boot setups
+setup-boot-windows-steam:
+    echo "Making efibootmgr -n accessible without sudo"
+    echo "%wheel ALL=(root) NOPASSWD: /usr/sbin/efibootmgr" | sudo tee /etc/sudoers.d/efibootmgr-config
+
```

Comment:
just syntax checking will be angry about the empty line IIRC

Reply:
I was not aware of that. I have removed the empty line from the script